10x_Experimental_Scenarios.md

```diff
@@ -7,87 +7,124 @@ The relationship between **biological samples** (e.g. a cell suspension extracte
+The `Index` column contains the oligo index IDs that can be looked up in [the `Features Table`][Library Features Table].
 
 # 10x Required CSVs
 
-### Sample Sheet CSV
+## Sample Sheet CSV
 | Lane| Sample | Index |
 |---|---|---|
 | 1  | S1_GEX | SI-GA-A3 |
 
-This table is in the format of the simple sample sheet for mkfastq. `Lane` refers to the 10x chip lane. `Sample` refers to our loading sample name (it will be appended with the BCL name automatically (I think) by mkfastq). `Index` refers to the oligos used to de-multiplex the BCL - in order to process custom oligos (e.g. ADT/HTO) we will pass the actual oligo index sequences in the `Index` column. 
+This table is in the format of the "simple samplesheet" consumed by `cellranger mkfastq`:
+- `Lane` refers to the 10x chip lane.
+- `Sample` refers to our loading sample name (the BCL name will be appended automatically by `mkfastq`; TODO: verify this).
+- `Index` refers to the oligos used to de-multiplex the BCL
+  - When processing custom oligos (e.g. ADT/HTO), pass the actual oligo sequences here.
 
-### Libraries CSV
-|  fastqs | sample  |  library_type |
+## Libraries CSV
+|  FASTQs | Sample  |  Library Type |
 |---|---|---|
 | /path/to/fastqs/ | S1_GEX_BCL-1 | Gene Expression |
 | /path/to/fastqs/ | S1_HTO_BCL-1 | Custom |
 
-`fastqs` give a path to the demultiplexed FASTQ files (cannot have comma-delimiited paths) - more than one path requres an additional row. `sample` is the sampe name assigned in the mkfastq simple sample sheet. `library_type` is self explanatory.
+- `FASTQs`: path to demultiplexed FASTQ files (cannot have comma-delimiited paths; more than one path requres an additional row).
+- `Sample` is the sample name assigned in the `mkfastq` simple samplesheet.
+- `Library Type` should be self-explanatory. <!-- Can we enumerate the possibilities here? Are they referenced anywhere else, or is this just a free-form "Notes" type of field? -->
 
-### Feature Reference CSV
+## Feature Reference CSV
 
 | id | name | read | pattern | sequence | feature_type | target_gene_id (optional) | target_gene_name (optional) |
 |---|---|---|---|---|---|---|---|
 |   |   |   |   |   |   |   |   |
+<!-- example row(s) -->
 
-This document is only necessary for feature barcode runs. `id` unique id for the feature (can't collide with gene name), `name` human readable feature name (e.g. gene name), `read` specifies which sequencing read contains the sequence (e.g. R2), `pattern` specifies how to extract seq from read, `feature_type` (e.g. custom), `target_gene_id` and `target_gene_name` are CRISPR specific and optional.
+This document is only necessary for feature barcode (FBM) runs:
+- `id`: unique id for the feature (can't collide with gene name)
+- `name`: human readable feature name (e.g. gene name)
+- `read`: specifies which sequencing read contains the sequence (e.g. R2)
+- `pattern`: specifies how to extract seq from read
+- `feature_type`: (e.g. custom <!-- more info? -->)
```

Comment:
more info about `feature_type`?

Reply:
I want to paraphrase the official docs (https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/feature-bc-analysis#feature-types)

`feature_type`: "Type of the feature. See the Library/Feature Types section for details on allowed values of this field. FASTQ data specified in the Library CSV File with a library_type that matches the feature_type will be scanned for occurrences of this feature. Each feature type in the feature reference must match a library_type entry in the Libraries CSV File. This field is case sensitive."

So we have the same options as we had for `library_type` in the `Libraries CSV` file.